src/components/data/DataTable.js

```diff
@@ -211,7 +211,8 @@ const DataTable = (props) => {
     });
 
   const getAllEntities = async () => {
-    const params = _.pickBy(_.trim, { pageSize: filteredCount, filterTerms: activeTextFilter, filterOperator });
+    const colFilt = !!columnFilter.filterColAttr && !!columnFilter.filterColTerm ? `${columnFilter.filterColAttr}=${columnFilter.filterColTerm}` : '';
```

Comment:
Maybe extract a method for the right-hand side of the assignment?  I notice the same logic on line 179.  This also might make space to expand the variable `colFilt` to something a bit more descriptive like 'columnFilterQueryString`.